### ISSUE TITLE
Game doesn't start on MacOS and app bundle icon doesn't show

### ISSUE THREAD

**Issue by Frotty** [libgdx/packr#173]
Hi @karlsabo ,
as I mentioned recently, my game stopped working on MacOS, the app shows up in the dock when I force-launch it via settings menu, but then just nothing happens. It used to work fine with the latest version of your fork and nothing game-wise really changed from what I know.
Updating packr to 3.0.0-SNAPSHOT did not resolve the issue.
Is there some debugging I can do to get some useful information on why this happens (it's an app bundle)?
Do I have to sign the app?
Also the icon I set is not shown for the app bundle.
Any help would be appreciated, thanks again for your efforts.

**Comment by karlsabo** [comment]
Have you tried launching the app from the command line and running in verbose mode?
`my.app/Contents/MacOS/myapp -c --verbose`

If it's a signing and notarization issue, there are some hotkeys to launch from the Finder to bypass macOS gatekeeper. A quick Google search should turn up some results. The packr readme has details for how to sign and notarize macOS applications, <https://github.com/libgdx/packr#macos-notarization-and-entitlements>.

**Comment by Frotty** [comment]
> Have you tried launching the app from the command line (terminal) and running in verbose mode?

Thanks, I  tried now. Perhaps this command could be added as a github issue template because I didn't find it in the readme.

The output:
```
$ ./oxrox.app/Contents//MacOS/oxrox -c --verbose
Changing working directory to /Users/frotty/Downloads/oxrox/oxrox.app/Contents/Resources ...
Loading JVM runtime library ...
fileSearch found libjli! filename=jre/lib/jli/libjli.dylib, lastFileSearch=
Loading libjli=/Users/frotty/Downloads/oxrox/oxrox.app/Contents/Resources/jre/lib/jli/libjli.dylib
Passing VM options ...
isZgcSupported()=1, hasJsonValue(jsonRoot, "useZgcIfSupportedOs", sajson::TYPE_TRUE)=0
  # -Xmx1G
  # -XstartOnFirstThread
Passing VM options:
  -Xmx1G
  -XstartOnFirstThread
Starting JVM on main thread (-XstartOnFirstThread found) ...
Creating Java VM ...
Passing command line arguments ...
Loading JAR file ...
Adding 1 classpaths ...
  # oxrox.jar
Invoking static de.fatox.desktop.DesktopLauncher.main() function ...
```

And then nothing happens (waited 10+ minutes).
The game shows in the dock and I can select it, but has no display.
Also the app bundle doesn't have the icon I set.

![Screenshot 2020-10-28 at 12 57 44](https://user-images.githubusercontent.com/1486037/97433054-2dea2e00-191d-11eb-9c10-8a2cdd48d28f.png)

![Screenshot 2020-10-28 at 12 58 00](https://user-images.githubusercontent.com/1486037/97433079-36daff80-191d-11eb-82e0-166d2c51b2dd.png)

> If it's a signing and notarization issue

I'm not doing any of that because it worked before without it. Do you think this would fix the problem or is it a different issue?

**Comment by karlsabo** [comment]
Packr is calling your main method, which is a good sign, do you have a log statement or System.out.println as the first line in your main? It would be nice to know if your Java code is running.

The CI tests run agains AdoptOpenJDK `OpenJDK8U-jre_x64_mac_hotspot_8u252b09.tar.gz`, the JDK should be fine. You might want to consider using the JRE unless there's something in the JDK that you need for your app.

For the icon, open up Contents/Info.plist and verify that it's correct. AFAIK the icon should work without signing or notarization.

**Comment by karlsabo** [comment]
@Frotty Did you get this working?

**Comment by Frotty** [comment]
No, sry. I only have access to a mac on occasion. I will try to test this with a startup sysout on monday!

**Comment by karlsabo** [comment]
@Frotty Have you gotten a chance to figure out the icon issue yet?

**Comment by karlsabo** [comment]
@Frotty what's the status of this issue?

**Comment by Frotty** [comment]
still need to test on a max 😬
